lib/bolt/target.rb

```diff
@@ -80,6 +80,22 @@ def to_h
       )
     end
 
+    def detail
+      {
+        'name' => name,
+        'uri' => uri,
+        'alias' => @target_alias,
```

Comment:
@target_alias is not an attribute for either Target or Target2. I think that for Target2 we can use the `target_alias` method, and we will need to introduce one for the `Target` object.